recipes/llvm/all/conanfile.py

```diff
@@ -0,0 +1,122 @@
+from conans import ConanFile, tools, CMake
+from conans.tools import Version
+from conans.errors import ConanInvalidConfiguration
+import os, shutil, glob
+
+projects = [
+    'clang',
+    'clang-tools-extra',
+    'compiler-rt',
+    'debuginfo-tests',
+    'libc',
+    'libclc',
+    'libcxx',
+    'libcxxabi',
+    'libunwind',
+    'lld',
+    'lldb',
+    'mlir',
+    'openmp',
+    'parallel-libs',
+    'polly',
+    'pstl'
+]
+
+default_projects = [
+    'clang',
+    'compiler-rt'
+]
+
+class Llvm(ConanFile):
+    name = 'llvm'
+    description = 'The LLVM Project is a collection of modular and reusable compiler and toolchain technologies'
+    url = 'https://github.com/conan-io/conan-center-index'
+    homepage = 'https://github.com/llvm/llvm-project'
+    license = 'Apache 2.0'
+    topics = 'conan', 'c++', 'compiler', 'tooling'
+
+    settings = 'os', 'arch', 'compiler', 'build_type'
+
+    no_copy_source = True
+    _source_subfolder = 'source_subfolder'
+
+    options = {**{ 'with_' + project : [True, False] for project in projects }, **{
+        'fPIC': [True, False]
+    }}
+    default_options = {**{ 'with_' + project : project in default_projects for project in projects }, **{
+        'fPIC': True
+    }}
+    generators = 'cmake_find_package'
+
+    @property
+    def repo_folder(self):
+        return os.path.join(self.source_folder, self._source_subfolder)
+
+    def project_folder(self, project):
+        return os.path.join(self.repo_folder, project)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = 'llvm-project-llvmorg-' + self.version
+        os.rename(extracted_dir, self._source_subfolder)
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version],
                       destination=self._source_subfolder, strip_root=True)
```
This will restart the build, which should be enough to make the error go away.